### PR TITLE
[fix #1009] Trying fix emojis in titles not working correctly.

### LINKED ISSUE #1009: emojis in titles not working correctly
## Bug Report

Processing the emojis in titles isn't working correctly. Sometimes just turning it into a single ":" character. Seems to work correctly if no leading space before the emoji code.

#### Steps to reproduce

Insert this content in a page
```
#### :arrow_forward:  :arrow_forward:
#### :arrow_forward::arrow_forward:
#### &nbsp;:arrow_forward: :arrow_forward:
#### &nbsp;:arrow_forward::arrow_forward:
#### &nbsp;:arrow_forward:&nbsp;:arrow_forward:
```

#### What is current behaviour
image

#### What is the expected behaviour
All emojis should appear

#### Other relevant information

- [x] Bug does still occur when all/other plugins are disabled?

- Your OS: mac
- Node.js version: 11.15
- npm/yarn version: 6.7
- Browser version: chrome latest
- Docsify version: cli is 4.4, using latest from cdn
- Docsify plugins: all disabled

### PR DESCRIPTION
**Pull request by Koooooo-7**
**Summary**
Trying fix #1009 emojis in titles not working correctly ,  

I attempt to  think all the types likes  `:words:` as an emoji code that I wont replace it to empty(`' '`).
At the same time, it causes another proplem that it cant solve `foo::ddd::ccc`  as u want ever, the same as the time `15:02:55`. BUT i think it is acceptable, because ：
A: In my opnion, we doesnt render the  `foo::ddd::ccc` 's title id correctly now.
B: I think it is a[ good idea](https://github.com/docsifyjs/docsify/issues/1009#issuecomment-581972542) to put them within a code backticks that we can distinguish the title ,the emojis and the configurations easily at now.

> Here's an idea - for the weird code syntax cases that #708 was trying to fix, require users to put them within a code backticks to avoid any conflicting treatment from config or emojis, for example:
>```## `MyPerlModule::Submodule::function` :emoji: :id=something ```

**TEST CASE**

- PASS
```
#### :arrow_forward: :arrow_forward:
#### :arrow_forward::arrow_forward:
#### &nbsp;:arrow_forward: :arrow_forward:
#### &nbsp;:arrow_forward::arrow_forward:
#### &nbsp;:arrow_forward:&nbsp;:arrow_forward:

# `MyPerlModule::Submodule::function` :dog: :dog: :id=something

### `15:22:10` 
# `foo::bar::baz`
# `foo:::bar:::baz`
```
- UNPASS
```
### 15:22:10
### foo::bar::baz
### foo:::bar:::baz
```

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

You have tested in the following browsers: (Providing a detailed version will be better.)

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

---

* [x] DO NOT include files inside `lib` directory.

### CODE REVIEW

**Comment by anikethsaha** [comment]
I will review it soon

File: src/core/render/compiler.js [hunk 1]
```diff
@@ -19,9 +19,12 @@ export function getAndRemoveConfig(str = '') {
     str = str
       .replace(/^'/, '')
       .replace(/'$/, '')
-      .replace(/(?:^|\s):([\w-]+)=?([\w-]+)?/g, (m, key, value) => {
-        config[key] = (value && value.replace(/&quot;/g, '')) || true
-        return ''
+      .replace(/(?:^|\s):([\w-]+:?)=?([\w-]+)?/g, (m, key, value) => {
+        if(key.indexOf(':') == -1){
```
Comment (anikethsaha) [inline src/core/render/compiler.js]:
```suggestion
        if(key.indexOf(':') === -1){
```

Linters !!!

**Comment by anikethsaha** [comment]
@Koooooo-7 Thanks a lot for your time and the fix. The fix looks good.

Just need to add notes in the docs for future ref !!!

File: src/core/render/compiler.js (hunk 1 above)
Comment (Koooooo-7) [reply src/core/render/compiler.js]: :sweat_smile: sry for that, I will fix it. :joy:

**Comment by Koooooo-7** [comment]
@anikethsaha 
I guess I didn't add the note in a wrong place . :dog:

**Pull request merged**